app/javascript/react/components/ReviewTile.js

```diff
@@ -1,11 +1,95 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
 
 const ReviewTile = (props) => {
+  const [votes, setVotes] = useState()
+  const [error, setError] = useState("")
+  useEffect(() => {
+    setError("")
+    let votePayLoad = {
+      reviewID: props.id
+    }
+    fetch('/api/v1/votes', {
```

Comment:
Post fetch request within the useEffect to run every time the component renders? If we need the vote count information when this ReviewTile loads we can send it along with the Review in props or make a GET request for the info